contracts/token/ControlledToken.sol

```diff
@@ -30,6 +30,7 @@ contract ControlledToken is ERC20Upgradeable, ControlledTokenInterface {
     initializer
   {
     __ERC20_init(_name, _symbol);
+    __ERC20Permit_init("PoolTogether Token");
```

Comment:
Would it be better to change to "PoolTogether ControlledToken" to ensure people are not confused with POOL?

Reply:
Yeah- I like that